### ISSUE TITLE
WCAG 2.2 rules

### ISSUE THREAD

**Issue by shunguoy** [IBMa/equal-access#1673]
**User story:**  
Create rules for testing new Success Criteria from WCAG 2.2 so customers who use our checker or tools can identify issues related to WCAG 2.2 requirements to make their products compliant with WCAG 2.2 

**Playback Demonstration:**
After the rules are implemented in the engine and released, the checker and tools that use the engine will contain rules that identify issues incompliant with WCAG 2.2 requirements. 
Additional unit test cases will be created to demonstrate the violations to the rules and the fixes to the issues.
**Help pages** will be provided to explain the requirements, and how to fix the violation if it occurs.

**Metric:**
The # of rules implemented that can automatically identify the issues related to WCAG 2.2 requirements.

**Comms:**
Communication will come out as part of the regular release cadence for the checker (a note in the What's New section).

## What project/section is the Epic for?

- Rules and rule help
- included in the checker and tools
- See squads Box note discussion [WCAG 2.2 rules](https://ibm.box.com/s/xmddww6efhqcowxlra7xrj0lprol1mvp)

## Definition of Done

- [x] Rules are implemented
- [x] Rule help content created
- [ ] All code committed to the main branch
- [x] Unit test cases are created for each rule
- [x] All unit tests pass (including accessibility)
- [x] Demo works in the staging environment
